### DISCUSSION TITLE
Does this project have an opinion on the similar Jumper XIAKE800?

### DISCUSSION THREAD

**Discussion by aviphysics** [StephenCarlson/MiniHawk-VTOL#46] [Q&A]
Hi,

Does this project have an opinion on the similar Jumper XIAKE800?
https://www.youtube.com/watch?v=gOlCFqfmsEU

Best,
Avi

**Comment by StephenCarlson** [comment]
@aviphysics I am still trying to form a proper and measured response; it has taken a lot of energy out of me at a time where I'm trying to get school work done. I saw your post also on the Hackaday site, I'll mirror my response there. 

Some primer on what I'll address, whenever I get around to writing it up:
- Summary of events from my perspective
- Short timeline of the MiniHawk-VTOL project
- Proof of my standing as the original designer and the project license
- Evidence of unsanctioned reuse, analysis of the Jumper design
- Elaboration of license requirements and terms
- Condensed version of my responses to this point
- Personal feelings and notes
- Steps and actions going forward

The Short Version: 
I didn't know that the MiniHawk had been adapted by Jumper and was/is being sold until someone on the Facebook VTOL Hangout Group posted it in the feed on 18 October 2022. Jumper never reached out (and still has not as of 25 Nov 2022) for collaboration or permission to use it, which is in violation of the Creative Commons License that is manifested in the Project Readme and on the Hackaday site. I myself have been trying to figure out how to mass-produce the MiniHawk for the same period of time that I've been developing it, but I clearly wasn't fast enough, and given the patched-together documentation and months-long gaps due to my workload, it might not have happened any time soon, but that does not justify the unsanctioned reuse of the design. If Jumper had merely been inspired by the design and had made their own design from scratch, there would be no contest from me, but they are very clearly using the MiniHawk geometry verbatim for several elements of their product. They had to redo a bunch of things, specifically the body and elevons, for it to be foam-moldable, but that does not absolve them of unfairly using the design without my permission.

I'm heartbroken and annoyed; having your design that you pour so much of your life into only to have it blatantly copied and sold without any recognition or compensation is something that happens to other people, right? But, I need not be a victim or a martyr; sharing this project has allowed many people to explore the world of VTOL, and with friends like Mark Appel and others who are fairly and correctly using elements of the MiniHawk to VTOL-ify other R/C planes, I'm happy that I have shared it and I hope I will not be scared to continue to share my designs and abilities. I've commented on every video I can find with the Jumper being reviewed, in order to set the record straight, and I otherwise am dependent on and thankful to the community for learning about what has happened and for also calling Jumper out.

**Comment by StephenCarlson** [comment]
A quick update:

It has been several weeks due to being a grad student with finals, then traveling to see family for winter break, and then diving right back into some interesting research that I hope will eclipse everything I've done to this point. But also, I must admit to some severe approach-avoidance syndrome, as I wish I could forget the drama of what has and is happening. Well, in reality, I've been putting in some effort on the situation, as follows:

- I've contacted GetFPV, RaceDayQuads, PyroDrone and other vendors that are within the sphere of legal effect and have preemptively warned them against ordering or reselling any of the Jumper clones or any future derivatives. For some of these, I've also renewed my query as to if any of these businesses would like to collaborate on mass producing this in an official partnership. 
- I replied to Jumper with a fiery rebuke, the text of which is rendered below.
- I've talked with a few folks in the community on the matter. 


# Emails
The email thread with Jumper is as follows. Some metadata: The initial email on the 29th was sent to the general email address for the business, but ended up being duplicated to the marketing account after Jumper reached out on Hackaday after a bunch of YouTube comments persuaded Jumper to contact me on Hackaday, and that contact on Hackaday referred me to the marketing address. All of the transaction below are to/from the marketing address, to give context for where I chew them out for them treating this as a marketing problem.

## What I sent Jumper on 29 Nov 2022:
> Jumper,
> 
> I am the designer of the MiniHawk-VTOL, which you used to make the Xaike800.
> 
> The MiniHawk-VTOL is released under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
> 
> I would have liked to have been contacted by you before you designed the Xaike800.
> 
> In comments on YouTube, you have stated that your engineers could not properly contact me, but that you are cooperating now. I am here.
> 
> Let's talk.
> 
> -Steve Carlson


## What Jumper sent me on 30 Nov 2022
> Hi Steve!
> 
> 
> Interesting! So you have been able. to see my comments? They disappear on my side for some reason!
> 
> Very nive to meet you!
> 
> Yes, we did in fact use your design and perfected it to be suitable for mass production. Sorry that it happened like this. I will be glad to communicate with you from now on though. Please, could you let me know how we can give you credit properly? What is your wish?
> 
> Best regards,
> Jumper RC Bushido


The text that I replied with on 11 Jan 2023 (below) had been pretty much written in the day or two after receiving the above response. I spent a few days mulling over the draft, but then final projects, end of semester, and everything else exploding shoved the whole matter out of my mind as I tried to focus on the higher priority stuff. With all that completed and over, I finally decided to send the draft reply, as follows. 

## What I replied with on 11 Jan 2023

>I am sorry that it happened like this too, I wish Jumper had contacted me before making the Xiake800. It took me many years to design the MiniHawk; I crashed many prototypes to get the aerodynamics right, and it took 10 years of prior experiments to know how to design it in the first place. I try to communicate the same way; with attention to detail and taking my time to get it right. It has taken me several weeks, and I am ready: Here are the issues that I would like to have resolved before continuing any business relationship or asking for credit:
> 
> 1. If I had 300000 subscribers on YouTube, would Jumper have contacted me before "perfecting" the MiniHawk design for commercial mass production? How popular did I need to be before I was treated fairly? It is clear to me that this is all being treated as a marketing problem, otherwise, why am I communicating to the marketing email address for Jumper?
> 2. The MiniHawk was released for free with the requirement that it not be mass produced without my collaboration, and any remix or copy is supposed to acknowledge me as the original designer, and such copies or remixes should be shared for free. Will Jumper release all the CAD files for the Xiake800 for free online, the same way the MiniHawk was released on GitHub and Hackaday?
> I cannot ask for money, even though the Creative Commons License allows me to demand such in Section 2, Part b, subsection 3, as follows:
> 
> > To the extent possible, the Licensor waives any right to collect royalties from You for the exercise of the Licensed Rights, whether directly or through a collecting society under any voluntary or waivable statutory or compulsory licensing scheme. In all other cases the Licensor expressly reserves any right to collect such royalties, including when the Licensed Material is used other than for NonCommercial purposes. 
> 
> 3. The reason I cannot ask for money is because I designed the MiniHawk using Autodesk Inventor under the Academic License, which does not allow me to make money from the things I design, and is only for research and free sharing of designs, which is what the MiniHawk is used for in my lab. I could buy a commercial license of Autodesk Inventor, and then I could use that to design a mass-production version of the MiniHawk that is new and somewhat different from what I released online, and therefore would not be under the Creative Commons License. I was actually planning on doing this at some point if I found someone I could team up with to mass produce the MiniHawk. If Jumper had reached out to collaborate with me first, I would have happily worked with Jumper to make a mass-produced version. I have other VTOL designs that I'm working on, if Jumper wants to team up to mass produce those, how can I trust Jumper?
> 
> -Steve Carlson


# Aftermath and Thoughts:

It has been a week, Jumper has not responded. But given I took a month to reply, so..... I'll check back here if I receive anything in the next few weeks?

I really want to forgive and forget this whole thing, but I don't want to enable or embolden Jumper or others like them to run with designs that are not suppose to be commercialized. The path forward for Jumper, if they want to salvage this, is going to require that they come clean and make amends. That means at least releasing their artwork in the same manner that the source artwork was and is presented, and crediting me as the source of most of the geometry that they are using, but given that their product is a bit of a lemon and appears to be getting cloned by others in China, I feel somewhat loathe to have my name and artistic integrity associated with the whole mess. Also, quick aside: While Jumper might claim that they were ignorant of the license, that is a flimsy excuse since it is in the Readme for the project both here on GitHub and the mirrored zip archives on Hackaday. 

# My Comments and Requests to the Community:

1. I'm sorry that I've taken so long to address this. Many have probably purchased the Xiake800 thinking that Jumper had reconciled with me. If you bought one without knowledge of the drama and issues, have fun with it; I want to share VTOL with everyone, even if this is an illegitimate product of my efforts. But now that you do know, if you have not purchased one, please don't, it is only enabling Jumper and others like them to get away with IP theft.
2. I'm sorry that I wasn't able to "beat Jumper to market" so to speak; I know lots of people want to have a cool foam VTOL and skip the 3D printing stage, I sure want to! If you or someone you know has the expertise to do a foam-molded R/C plane kit, please get in contact with me, I'm still looking for and trying to figure out how to partner with someone one that can "do it right". 
3. Pressure Jumper to release all the Xiake800 manufacturing artwork, so as to at least satisfy the Share-Alike clause of the Creative Commons License that the MiniHawk is released under. That means at least the STLs for everything, but also possibly X_T and/or STEP versions of at least the canopy and nacelles. They actually have satisfied this so far as the canopy is concerned [ArduPilot Discuss Thread](https://discuss.ardupilot.org/t/jumper-xiake-800-flight-controller/93380/39), lets see them release all of it.
4.  Like, Share and Subscribe to my [YouTube channel](https://www.youtube.com/channel/UCFDnmh8lvr5YPe9E5G-kyJA)? I can't believe I just said that. Whatever, I'm just curious if my 300k subscriber statement in my response email might one day be realized? Share the MiniHawk-VTOL, try printing it, keep me stimulated with feedback and I will try to improve it.
5. Make it known that Jumper reused my design without my consent and against the rules. Best analogy: Imagine that you are an artist, and you draw a really cool airplane drawing and post it to ArtStation or Instagram. Then one day you see your drawing on a kid's lunch box, and come to find out that someone in China saw your picture and decided to use it without telling you and is currently selling lots of lunch boxes with your drawing. That is my life right now.


-Steve

**Reply by jhiggason** [reply]
Hey man- I know this must be heartbreaking- I can't even imagine after all this work having this happen.  

  Just know you have a community of people that are trying our best to support you.  I'm posting when I see people recommend Jumper products every chance I get.  I just link them to this discussion and tell them not to buy stuff from a company that is blatantly ripping off a completely open source design.  I've also tried emailing GetFPV and RaceDayQuads about the Jumper products on their websites and how they are OK supporting a company that is ripping off other vendors.   RaceDayQuads doesn't seem to be restocking their Jumper stuff...so maybe??????

If you are guys are looking for good Parts companys that ship fast to the US.  Try RotorGeeks based out of Canada.  They do fast USPS shipping.  They don't carry any Jumper Products. =) =)  =)  These guys have been very helpful everytime I had to email them and seem to support good products =)

**Comment by jhiggason** [comment]
Hey Steve-  I was thinking about your problem- could you contact Autodesk?  I bet their legal department has WAY bigger resources for lawyers than you do- especially since you used their free version of the software and then Jumper took that free design and now is making money on it violating Autodesk's licensing.  Autodesk surely won't stand for this......Just another thought no hard feelings if you dont think this will work. 

-Jeff

**Reply by aviphysics** [reply]
I expect that Jumper would have completely remodeled the project to make it work for injection molded foam, and either did it with a professional license in whatever software or they super don't give a fk.

**Reply by aviphysics** [reply]
It is more like there would have to be some substantial open source union that we all bought into to secure a legal fund to defend our projects against this kind of thing.

**Comment by jhiggason** [comment]
If you can take a moment to write out something on my post it will give it legitimacy.  

https://www.reddit.com/r/drones/comments/117dlje/do_your_best_to_not_support_the_chinese_company/

**Comment by jhiggason** [comment]
Have you had any progress on this? At least the design isn't on their website any longer and I only see one vendor who is even selling it anymore. (Another Chinese company that I won't link to)

Anytime I see someone recommend a Jumper product- I try to steer them to a different manufacturer and reference this page. I know it's not much- but folks like you- who put their heart and soul into opensource work- deserve better than this.

This design is next on my build list- I'll do my best to promote you when I can. =)

![It aint much!](https://i.kym-cdn.com/entries/icons/original/000/028/021/work.jpg)
